yellow/definition-of-quality-gates.md

```diff
@@ -1,16 +1,18 @@
-# Definition of quality gates
+# Definition of Quality Gates
 
-Quality gates for found vulnerabilities are defined. Provide feedback about vulnerabilities and time of exposure.
+Define quality gates adhering to your companies policy, make sure that all requirements are fulfilled. Also define quality gates for found vulnerabilities and provide feedback about vulnerabilities and time of exposure.
```

Comment:
```suggestion
Define quality gates adhering to your companies policy. Make sure that all requirements are fulfilled. Also define quality gates for found vulnerabilities and provide feedback about vulnerabilities and time of exposure.
```

Reply:
Can you explain which requirements are addressed here.